smart_contracts.rst

```diff
@@ -337,6 +461,35 @@ Unlocks all pending transfers by providing the entire merkle tree of pending tra
     ``unlock`` must be called after ``settleChannel`` because it needs the ``locksroot`` from the latest :term:`balance proof` in order to guarantee that all locks have either been unlocked or have expired.
 
 
+.. _settle-computation:
+
+Computation of balances
+~~~~~~~~~~~~~~~~~~~~~~~
+
+There is an elaborate algorithm to safely compute the balances, avoiding simple errors like over- and underflows as well as vulnerabilities for cheating.
+A possible attempt at cheating would be to present an outdated balance proof before an unlocking event, so that the locked amount would be assessed too high.
```

Comment:
I would just resume to the settlement constraints and leave the analysis about cheating out of here for now. I don't think the security analysis of the settlement algorithm should be in the spec.
Also, `A possible attempt at cheating` might be interpreted wrongly.

Reply:
I removed the whole commit related to issue #86 from this pr since pr #99 covers it.